shopelectro/settings/base.py

```diff
@@ -215,6 +215,8 @@
 }
 
 SELENIUM_URL = os.environ.get('SELENIUM_URL', 'http://selenium:4444/wd/hub')
+SELENIUM_WAIT_SECONDS = int(os.environ.get('SELENIUM_WAIT_SECONDS', 60))
+SELENIUM_TIMEOUT_SECONDS = int(os.environ.get('SELENIUM_TIMEOUT_SECONDS', 30))
```

Comment:
we also need to set them to `docker/env_files/app`.
And to `.drone.yml` as secrets. See secrets using the example at `docker-push` drone.yml section.

To check yourself take rm default values. Code example: `os.environ.get['SELENIUM_TIMEOUT_SECONDS']`.
Then check if test launches at CI and in localhost machine